CHANGES.md

```diff
@@ -2,6 +2,10 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.168 Jan 10 2022
+- Fix description of various API attributes
```

Comment:
what about "WIP: adding field to hold validation error message" or you're not including it on purpose?

Reply:
I think that was merged after this MR was created, be awesome to include it though :pray:

Reply:
rebased